docs/INSTALL_OMNIA_CONTROL_PLANE.md

```diff
@@ -121,7 +121,7 @@ git clone -b release https://github.com/dellhpc/omnia.git
 | 	timezone	                                                                         | 	**GMT**	                                           |  Enter a timezone that is set during the   provisioning of OS. **GMT** is set as the default time zone. You can set the   time zone to **EST**, **CET**, **MST**, **CST6CDT**, or **PST8PDT**. For a   list of available time zones, see the `appliance/common/files/timezone.txt`   file.	                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | 	language	                                                                         | 	**en-US**	                                         |  Set the language used during the   provisioning of OS. By default, it is set to **en-US**.	                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| 	iso_file_path	[Required]	                                                          | **/root/Rocky-8.4-x86_64-minimal.iso**		                                                  |  Provide the CentOS-7-x86_64-Minimal-2009 ISO   file path. This ISO file is used by Cobbler to provision the OS on the   compute nodes. **Note**: It is recommended that the ISO image file is not   renamed. And, you **must not** change the path of this ISO image file as the   provisioning of the OS on the compute nodes may be impacted.	                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
We are supporting both Rocky 8.4 and CentOs 7.9 on the Head/Compute/Login/NFS Node.